### PR TITLE
feat: lists completed

### PR DESCRIPTION
**Pull request by sonishreyas**
## Change Logs
Feature: lists added into the component library

## Screenshots
image

## Preview Link
https://deploy-preview-13--ripple-ui.netlify.app/components/lists/lists.html

### CODE REVIEW

**Review by yashpatel1797 — APPROVED** [review]
Looks good.

File: css/lists.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.stacked-list .basic-card {
+	display: flex;
+	align-items: center;
+	gap: 1rem;
```
Comment (Prabhav-Sharma) [inline css/lists.css:4]: Use global variables for spacing for properties like margin, padding, and gap to maintain consistency across the website

Comment (soni-shreyas) [reply css/lists.css:4]: Thanks will make that change after the assignment is complete

Comment (sonishreyas) [reply css/lists.css:4]: Sure will make that change after finishing assignment

**Pull request merged**